apps/src/templates/sectionProgress/progressTables/progressTableHelpers.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+
+// styles rows: dark backgrounds, primary rows, expanded rows
```

Comment:
NIT: "styles rows"? how about "row styles"?

Reply:
Resolving, since this comment was removed.